### PR TITLE
Jetpack Backup: Add Backup retention management setting

### PR DESCRIPTION
**Pull request by Initsogar**
#### Proposed Changes

* Add a new `Days of backups saved` (backup retention period) setting in Jetpack Cloud, under Settings page. For this PR, it will be available only for A12s (dev & stage environment). The idea is to start the review process of this new functionality.
* This iteration includes:
  * Allow to update the setting only to a number of days that doesn't require a storage upgrade.
  * Fetch and update the customer-defined retention period to our backend.
  * Confirmation dialog when attempting to update to a lower retention period.

#### Demo

https://user-images.githubusercontent.com/1488641/218225112-ce96783e-d079-447b-8df2-24d9f54666fd.mov

↪ [Demo v1 (deprecated)](https://user-images.githubusercontent.com/1488641/217998763-cc89187c-9ce8-4536-a8fb-785c153ab76c.mov)

#### Screenshots

| Description | Screenshot |
|---|---|
| Setting page | image |
| Confirmation dialog | image |
| Tooltip | image |
| Success notice | image |
| Error notice | image |

#### Testing Instructions

* Start a Jetpack Cloud live branch
* Navigate to Settings and wait for the new setting to load.
* The `Update settings` button will be disabled when the current plan or an option that requires an upgrade are selected.
* You could try to update the days of backups saved. The way to re-confirm it was properly saved is refreshing the page.

Notes:
* If you haven't configured this setting before, the current retention period associated to your plan will be preselected. Otherwise, your current configuration will be preselected.

#### Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by grbicsanja** [comment]
This is looking great! 🙌
Can we please have the confirmation notice say:
`You have successfully changed the time period of saved backups to X days`

File: client/components/backup-retention-management/hooks.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import { useMemo } from '@wordpress/element';
+import { useSelector } from 'react-redux';
+import { useStorageText } from 'calypso/components/backup-storage-space/hooks';
+import getBackupCurrentSiteSize from 'calypso/state/rewind/selectors/get-backup-current-site-size';
+import getRewindBytesAvailable from 'calypso/state/rewind/selectors/get-rewind-bytes-available';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { RetentionRadioOptionType } from './consts';
```
Comment (rustamshaikh) [inline client/components/backup-retention-management/hooks.tsx]:
```suggestion
import type { RetentionRadioOptionType } from './consts';
```
Since we are importing type.

File: client/components/backup-retention-management/hooks.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+/**
+ * Estimate the current site size based on the last backup size, adding a 25% buffer
+ */
+export function useEstimatedCurrentSiteSize(): number {
+	const siteId = useSelector( getSelectedSiteId ) as number;
+	const lastBackupSize = useSelector(
+		( state ) => getBackupCurrentSiteSize( state, siteId ) as number
+	);
+	return useMemo( () => lastBackupSize + lastBackupSize * 0.25, [ lastBackupSize ] );
```
Comment (rustamshaikh) [inline client/components/backup-retention-management/hooks.tsx]: `0.25` - Maybe we can define it as a constant?

File: client/components/backup-retention-management/consts.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import type { TranslateResult } from 'i18n-calypso';
+
+export const RetentionOptions = {
+	RETENTION_DAYS_7: 7,
+	RETENTION_DAYS_30: 30,
+	RETENTION_DAYS_120: 120,
+	RETENTION_DAYS_365: 365,
+} as const;
+
+export interface RetentionRadioOptionType {
```
Comment (rustamshaikh) [inline client/components/backup-retention-management/consts.tsx]: Maybe we can move this type `RetentionRadioOptionType` to a 'type only' file say `types.ts`

**Comment by grbicsanja** [comment]
The info icon and tooltip is missing. [See design here](https://www.figma.com/file/zEkRPQwNT9aykpCyJvrSt5/Allow-customers-to-define-their-own-backup-retention?node-id=3147%3A38368&t=u5jSmMRkhRo6u8Sm-4).
Screenshot 2023-02-10 at 13 26 14

**Comment by grbicsanja** [comment]
I see the warning prompt even when I'm choosing a longer time period. It should only be shown when I am reducing the time and loosing saved backups.

Here in this screenshot, I was moving from 7 days to a year.
Screenshot 2023-02-10 at 13 27 26

File: client/components/backup-retention-management/retention-options/retention-options-control.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState, useCallback } from '@wordpress/element';
+import React from 'react';
+import { RetentionRadioOptionType } from '../consts';
```
Comment (rustamshaikh) [inline client/components/backup-retention-management/retention-options/retention-options-control.tsx]: `import type` - Same w.r.t import type and separating constants from types.

File: client/components/backup-storage-space/hooks.tsx [hunk 5]
```diff
@@ -111,3 +111,49 @@ export const useDaysOfBackupsSavedText = (
...
+					if ( unitAmount % 1 === 0 ) {
+						return translate( '%(storageInBytes)dTB', {
+							args: { storageInBytes: unitAmount },
+							comment: 'Must use unit abbreviation; describes an storage amounts (e.g., 1TB)',
+						} );
+					}
+
+					return translate( '%(storageInBytes).1fTB', {
+						args: { storageInBytes: unitAmount },
+						comment: 'Must use unit abbreviation; describes an storage amounts (e.g., 1.5TB)',
```
Comment (rustamshaikh) [inline client/components/backup-storage-space/hooks.tsx]:
```suggestion
						comment: 'Must use unit abbreviation; describes an storage amounts with 1 decimal point (e.g., 1.5TB)',
```
Nitpicking, but it may be helpful to translators.

File: client/jetpack-cloud/sections/settings/controller.tsx [hunk 6]
```diff
@@ -22,19 +24,24 @@ export const advancedCredentials: PageJS.Callback = ( context, next ) => {
...
-					siteId={ siteId }
-					trueComponent={ sectionElt }
-					falseComponent={ <NoSitesPurchasesMessage /> }
-					loadingComponent={ <AdvancedCredentialsLoadingPlaceholder /> }
-				/>
-			}
-			loadingComponent={ <AdvancedCredentialsLoadingPlaceholder /> }
-		/>
+		<>
+			{ config.isEnabled( 'jetpack/backup-retention-settings' ) ? (
```
Comment (rustamshaikh) [inline client/jetpack-cloud/sections/settings/controller.tsx:28]: Great!

**Review by rustamshaikh — COMMENTED** [review]
Great work @Initsogar 👍 Tested it from Jetpack Cloud live and works as expected. Left some feedback. Let me know what you think.

**Comment by rustamshaikh** [comment]
In addition to what @grbicsanja said, these are some UI-related differences that might have been missed (in any case forgive me I am not a good designer if any 🙂)

The height of the loading button:
Screenshot 2023-02-10 at 6 51 34 PM

The spacings which don't match with Design P2 are highlighted (with <--) (Uploaded Design P2 screenshot for simplicity).

Screenshot 2023-02-10 at 6 43 08 PM

File: client/components/backup-storage-space/hooks.tsx (hunk 5 above)
Comment (Initsogar) [reply client/components/backup-storage-space/hooks.tsx]: Done! Thanks for the suggestion.

File: client/components/backup-retention-management/hooks.tsx (hunk 2 above)
Comment (Initsogar) [reply client/components/backup-retention-management/hooks.tsx]: Added it to the constants.ts file, thanks!

File: client/components/backup-retention-management/consts.tsx (hunk 3 above)
Comment (Initsogar) [reply client/components/backup-retention-management/consts.tsx]: I agree. Just moved it to a new types.ts file.

**Comment by Initsogar** [comment]
Thanks @grbicsanja and @rustamshaikh, just addressed all your comments and suggestions:
* Added tooltip. Also included it on the PR description, so we can double check the wording there.
* Updated success notice text to `You have successfully changed the time period of saved backups to X days`
* Update confirmation dialog to only be required when updating to a lower retention period
* Adjusted translations comments
* Refactored types and constants
* Adjusted spacing between elements

**Review by rustamshaikh — APPROVED** [review]
Verified the fixes and it looks great to me 🎉 - Thanks @Initsogar for adding more tests 👍

File: client/components/backup-storage-space/hooks.tsx [hunk 7]
```diff
@@ -111,3 +111,50 @@ export const useDaysOfBackupsSavedText = (
...
+/**
+ * The idea is to convert any storage amount in bytes to a human readable format.
+ *
+ * @param storageInBytes The storage amount in bytes
+ * @returns				 The storage amount in a human readable format
+ */
+export const useStorageText = ( storageInBytes: number ): TranslateResult | string => {
+	const translate = useTranslate();
+
+	return useMemo( () => {
```
Comment (tyxla) [inline client/components/backup-storage-space/hooks.tsx:124]: Is the memoization here worth the effort? What's the performance impact with and without it?

File: client/components/backup-retention-management/types.ts [hunk 8]
```diff
@@ -0,0 +1,10 @@
+import type { TranslateResult } from 'i18n-calypso';
+
+export interface RetentionRadioOptionType {
+	label: string; // 7 days
+	spaceNeeded: TranslateResult | string; // 700MB
+	upgradeRequired: boolean;
+	isCurrentPlan: boolean;
+	value: number; // 7
```
Comment (tyxla) [inline client/components/backup-retention-management/types.ts:8]: Is this comment required or incomplete perhaps?

```suggestion
	value: number;
```

File: client/components/backup-retention-management/loading/index.tsx [hunk 9]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
```
Comment (tyxla) [inline client/components/backup-retention-management/loading/index.tsx]: Should we import only the `React.FC` type if we really need that only?

File: client/components/backup-retention-management/loading/style.scss [hunk 10]
```diff
@@ -0,0 +1,16 @@
+.backup-retention-management__loading {
+	&.loading {
```
Comment (tyxla) [inline client/components/backup-retention-management/loading/style.scss:2]: If this adds an extra `loading` class, why do we need the `__loading` suffix for `backup-retention-management__loading` ?

File: client/components/backup-retention-management/info-tooltip/index.tsx [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+						onClick={ closeTooltip }
+					>
+						<Gridicon icon="cross" size={ 18 } />
+					</Button>
+				</p>
+				<hr />
+				{ translate( '{{a}}Learn more…{{/a}}', {
+					components: {
+						a: (
+							<a
```
Comment (tyxla) [inline client/components/backup-retention-management/info-tooltip/index.tsx:57]: Could use `<ExternalLink />` here.

File: client/components/backup-retention-management/info-tooltip/index.tsx [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+					<Button
+						borderless
+						compact
+						className="retention-setting-info-tooltip__close-tooltip"
+						onClick={ closeTooltip }
+					>
+						<Gridicon icon="cross" size={ 18 } />
+					</Button>
+				</p>
+				<hr />
```
Comment (tyxla) [inline client/components/backup-retention-management/info-tooltip/index.tsx:53]: Using `hr`s is a suboptimal way to lay out the presentation. Could be a better bet to add an extra wrapper and use CSS.

File: client/components/backup-retention-management/info-tooltip/index.tsx [hunk 13]
```diff
@@ -0,0 +1,69 @@
+import { Button, Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import * as React from 'react';
```
Comment (tyxla) [inline client/components/backup-retention-management/info-tooltip/index.tsx]: Do we need all of `react` to be imported?

File: client/components/backup-retention-management/retention-options/retention-option-card.tsx [hunk 14]
```diff
@@ -0,0 +1,75 @@
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
```
Comment (tyxla) [inline client/components/backup-retention-management/retention-options/retention-option-card.tsx]: Same on importing all of `react`

File: client/components/backup-retention-management/retention-options/retention-option-card.tsx [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+import type { TranslateResult } from 'i18n-calypso';
+
+interface RetentionOptionCardProps {
+	label: string;
+	spaceNeeded: TranslateResult | string;
```
Comment (tyxla) [inline client/components/backup-retention-management/retention-options/retention-option-card.tsx:8]: Won't the `TranslateResult` include `string` already?

Also, it's an odd prop name - it sounds like a boolean but it's not. Thoughts?

File: client/components/backup-retention-management/retention-options/retention-option-card.tsx [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+
+	return (
+		<div
+			className="retention-option"
+			role="radio"
+			aria-checked={ checked }
+			onClick={ handleOptionClick }
+			onKeyDown={ handleOptionClick }
+			tabIndex={ 0 }
+		>
```
Comment (tyxla) [inline client/components/backup-retention-management/retention-options/retention-option-card.tsx:39]: Any more accessible way to compose a radio button? We have a bunch of radio control components available for use.

File: client/components/backup-retention-management/retention-options/retention-options-control.tsx [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+		}
+	}, [ selectedOption, onChange ] );
+
+	const handleRetentionOptionChange = useCallback( ( value: number ) => {
+		setSelectedOption( value );
+	}, [] );
+
+	return (
+		<div className="retention-form__options">
+			{ Object.values( retentionOptions ).map( ( option ) => (
```
Comment (tyxla) [inline client/components/backup-retention-management/retention-options/retention-options-control.tsx:31]: If we're going to use `retentionOptions` as an array of values, should we reform the component to expect an array and use that instead of calling `Object.values()`  on an object every time?

File: client/components/backup-retention-management/index.tsx [hunk 18]
```diff
@@ -0,0 +1,239 @@
...
+
+	useEffect( () => {
+		if ( updateRetentionRequestStatus === BACKUP_RETENTION_UPDATE_REQUEST.SUCCESS ) {
+			// Update the current retention plan to the one updated by the user.
+			setCurrentRetentionPlan( retentionSelected );
+			setConfirmationDialogVisible( false );
+		} else if ( updateRetentionRequestStatus === BACKUP_RETENTION_UPDATE_REQUEST.FAILED ) {
+			setConfirmationDialogVisible( false );
+		}
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (tyxla) [inline client/components/backup-retention-management/index.tsx:159]: Why is this necessary?

File: client/components/backup-retention-management/index.tsx [hunk 19]
```diff
@@ -0,0 +1,239 @@
...
+			currentRetentionPlan,
+			retentionSelected === RETENTION_OPTIONS.RETENTION_DAYS_120
+		),
+		[ RETENTION_OPTIONS.RETENTION_DAYS_365 ]: usePrepareRetentionOptions(
+			translate( '1 year' ),
+			RETENTION_OPTIONS.RETENTION_DAYS_365,
+			currentRetentionPlan,
+			retentionSelected === RETENTION_OPTIONS.RETENTION_DAYS_365
+		),
+	};
```
Comment (tyxla) [inline client/components/backup-retention-management/index.tsx:103]: Do we really need to redeclare those on every render?

File: client/components/backup-retention-management/constants.ts [hunk 20]
```diff
@@ -0,0 +1,11 @@
+export const RETENTION_OPTIONS = {
+	RETENTION_DAYS_7: 7,
```
Comment (tyxla) [inline client/components/backup-retention-management/constants.ts:2]: Is there a good reason to have those declared in an object? They seem pretty simple and used only in one place.

File: client/components/backup-retention-management/hooks.tsx [hunk 21]
```diff
@@ -0,0 +1,77 @@
...
+ * @param desiredRetentionDays   Desired backup retention days
+ * @returns Space needed in bytes
+ */
+export function useEstimateSpaceNeeded(
+	currentSiteSizeInBytes: number,
+	desiredRetentionDays: number
+): number {
+	return useMemo( () => {
+		return currentSiteSizeInBytes * desiredRetentionDays;
+	}, [ currentSiteSizeInBytes, desiredRetentionDays ] );
```
Comment (tyxla) [inline client/components/backup-retention-management/hooks.tsx:37]: Have we evaluated the need for each of those memoizations? What's the performance impact and why do we need them? Can we do without them? Would be good to measure and use that to decide whether we need them.

**Review by tyxla — COMMENTED** [review]
Hey folks! This is a pretty big PR, have you considered splitting it into smaller pieces? Usually, we recommend such bigger PRs to land in smaller places because that allows us to thoroughly test and review, and ensure that we don't introduce regressions. The more the code, the more difficult it is to catch a bug or regression.

Furthermore, I've left some drive-by feedback and questions.

Please let me know if you're interested in me reviewing a particular aspect of the PR.

Thanks 🙌

File: client/components/backup-retention-management/index.tsx (hunk 18 above)
Comment (Initsogar) [reply client/components/backup-retention-management/index.tsx:159]: @tyxla in this case, I would like this check to occur only when `updateRetentionRequestStatus` changes. If I don't include this line, the linter will keep expecting to pass all as dependencies.

Comment (tyxla) [reply client/components/backup-retention-management/index.tsx:159]: Technically, the rest of the dependencies can also change and that theoretically should re-run the effect. Is there a good reason to seek workarounds for this default behavior?

File: client/components/backup-retention-management/info-tooltip/index.tsx (hunk 13 above)
Comment (Initsogar) [reply client/components/backup-retention-management/info-tooltip/index.tsx]: You are right! Just fixed it.

File: client/components/backup-retention-management/retention-options/retention-option-card.tsx (hunk 14 above)
Comment (Initsogar) [reply client/components/backup-retention-management/retention-options/retention-option-card.tsx]: Fixed!

File: client/components/backup-retention-management/loading/index.tsx (hunk 9 above)
Comment (Initsogar) [reply client/components/backup-retention-management/loading/index.tsx]: Fixed!

**Comment by Initsogar** [comment]
> Hey folks! This is a pretty big PR, have you considered splitting it into smaller pieces? Usually, we recommend such bigger PRs to land in smaller places because that allows us to thoroughly test and review, and ensure that we don't introduce regressions. The more the code, the more difficult it is to catch a bug or regression.
> 
> Furthermore, I've left some drive-by feedback and questions.

@tyxla thanks for all this valuable feedback! Indeed, it has been quite complicated to make this PR smaller. We tried to add some dependencies in separate PRs in #73031, #73108 and #73007, but I agree that we could do better.

As we need to continue iterating on this new component, I would appreciate it if we could merge this PR and address the comments and suggestion in a new PR. What do you think?

**Comment by tyxla** [comment]
> @tyxla thanks for all this valuable feedback! Indeed, it has been quite complicated to make this PR smaller. We tried to add some dependencies in separate PRs in #73031, #73108 and #73007, but I agree that we could do better.
> 
> As we need to continue iterating on this new component, I would appreciate it if we could merge this PR and address the comments and suggestion in a new PR. What do you think?

I really appreciate the effort. Ideally, we should approach changes with the atomic change in mind, and keep large PRs like this only for testing or prototyping. I realize it's hard and time-taking sometimes, but it's worth the effort and our future selves will thank us for that. Anyway, I won't be blocking this PR if you can commit to addressing the feedback in subsequent PRs in the shorter term.

**Comment by Initsogar** [comment]
> I really appreciate the effort. Ideally, we should approach changes with the atomic change in mind, and keep large PRs like this only for testing or prototyping. I realize it's hard and time-taking sometimes, but it's worth the effort and our future selves will thank us for that. Anyway, I won't be blocking this PR if you can commit to addressing the feedback in subsequent PRs in the shorter term.

Thank you, @tyxla, I commit to address the feedback in a new PR in short term.

File: client/components/backup-retention-management/index.tsx (hunk 18 above)
Comment (Initsogar) [reply client/components/backup-retention-management/index.tsx:159]: I think I see now what's the issue and how it should be.

After updating the retention period, I'm setting the `retentionSelected` at that moment as the `currentRetentionPlan`, which is not the best approach. The right way should be to update the retentionPlan on the redux state using an action dispatcher in the [onSuccess callback](https://github.com/Automattic/wp-calypso/pull/73035/files#diff-d9ef76a91dcc2d8e7451ccddfbc3fc6b732bba3c09dc25de5f55da431b361c2fR24).

I'll take a double look and handle it in a new PR.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7814445

Thank you @Initsogar for including a screenshot in the description! This is really helpful for our translators.

File: client/components/backup-retention-management/index.tsx (hunk 18 above)
Comment (Initsogar) [reply client/components/backup-retention-management/index.tsx:159]: Just [submitted a PR](https://github.com/Automattic/wp-calypso/pull/73300) for review related to this, it would be great if you can take a look :) https://github.com/Automattic/wp-calypso/pull/73300

File: client/components/backup-retention-management/constants.ts (hunk 20 above)
Comment (Initsogar) [reply client/components/backup-retention-management/constants.ts:2]: Double-checked and no. It was an attempt to declare what are the options that we are going to use. I removed this and instead, added the `RetentionPeriod` from `calypso/state/rewind/retention/types` to validate that we use valid values.

Addressed in #73303

File: client/components/backup-retention-management/retention-options/retention-options-control.tsx (hunk 17 above)
Comment (Initsogar) [reply client/components/backup-retention-management/retention-options/retention-options-control.tsx:31]: Agree, it makes sense. We just prepared a refactor for this.

Addressed in #73303

File: client/components/backup-retention-management/index.tsx (hunk 19 above)
Comment (Initsogar) [reply client/components/backup-retention-management/index.tsx:103]: I think it makes more sense to have it as a component state. We just refactored it.

Addressed in #73303

File: client/components/backup-retention-management/retention-options/retention-option-card.tsx (hunk 15 above)
Comment (Initsogar) [reply client/components/backup-retention-management/retention-options/retention-option-card.tsx:8]: > Won't the TranslateResult include string already?

It seems [it is actually defined as a React.ReactChild](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/types/index.d.ts#L59) and Typescript doesn't like the assumption that it is a string.

Either way, we revisited it and noticed that it is preferable to pass the space needed in bytes, so we made a little refactor there.

Addressed in #73303

File: client/components/backup-retention-management/types.ts (hunk 8 above)
Comment (Initsogar) [reply client/components/backup-retention-management/types.ts:8]: Incomplete 🥲. The 7 was actually an example.

Either way, [we refactored it a bit](https://github.com/Automattic/wp-calypso/pull/73303/files#diff-71e240ea9b6bd13fea335936ce41b9d60fc06babcbc766ce1ed40cde9f4905e5), including a `RetentionPeriod` type instead of `number`, so we can enforce to only use the valid values (7, 30, 120, 365 for now).

Addressed in #73303

File: client/components/backup-retention-management/hooks.tsx (hunk 21 above)
Comment (Initsogar) [reply client/components/backup-retention-management/hooks.tsx:37]: We did some refactors on the way we handle the retention options, and we needed to stop using hooks, as they must only be called at top level.

Either way, at first sight and given that they are simple math operations and the fact that we are not calling too many times those functions, the memoization is not required for `useEstimatedCurrentSiteSize()` and `useEstimateSpaceNeeded()`.

Addressed in #73303

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7814445

Thank you @Initsogar for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7814445

Thank you @Initsogar for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7814445).
